app/src/main/java/com/example/atheneum/viewmodels/FirebaseRefUtils/DatabaseWriteHelper.java

```diff
@@ -211,6 +212,11 @@ public void onComplete(@Nullable DatabaseError databaseError, @NonNull DatabaseR
         });
     }
 
+
+    public static void deleteRequest(Request request){
+        RequestCollectionRefUtils.getBookRequestCollectionRef(request.getBookID()).setValue(null);

```

Comment:
There's also an owner request collection as well? Will this delete the request there as well? What do you think @MarcusBoay?

Reply:
Just realized this is incorrect, should be owner request collection, not book request collection. I will fix this.